web/src/portico/header.ts

```diff
@@ -73,11 +73,6 @@ $(() => {
         on_tab_menu_selection_change();
     });
 
-    $("body").on("change", "top-menu-tab-input", (e) => {
-        $("#top-menu-tab-close").prop("checked", true);
-        on_tab_menu_selection_change(e);
-    });
-
```

Comment:
@amanagr can you investigate what this is supposed to be and submit a follow-up PR as appropriate? Looks like it's from d564025d6a67d7ea36fbd7494754452b7eccb045.

Reply:
```
    $(".top-menu-tab-input").on("click", function (this: HTMLElement) {
        on_tab_menu_selection_change(this);
    });
```

Does the correct thing. So this was dead code. Sorry!

Reply:
```
._full-height-no-scroll {
        height: 100%;
        overflow: hidden;
    }
```

Seems like it is trying to prevent body from scrolling when navbar is occupying the full height on mobile width. Since navbar already captures the full width and height on mobile and also captures any scroll event event, doesn't look like we need to have this additional fix.